iphone/Maps/Classes/CustomViews/MapViewControls/TrafficButton/MWMTrafficButtonViewController.mm

```diff
@@ -128,11 +127,11 @@ - (void)refreshAppearance
     break;
   case MWMTrafficManagerStateExpiredData:
     btn.imageName = @"btn_traffic_on";
-    [MWMToast showWithText:L(@"traffic_update_maps_text")];
+    [[MWMToast toastWithText:L(@"traffic_update_maps_text")] show];
     break;
   case MWMTrafficManagerStateExpiredApp:
     btn.imageName = @"btn_traffic_on";
-    [MWMToast showWithText:L(@"traffic_update_app_message")];
+    [[MWMToast toastWithText:L(@"traffic_update_app_message")] show];
```

Comment:
Swift method 'Toast.toast(withText:)' uses '@objc' inference deprecated in Swift 4; add '@objc' to provide an Objective-C entrypoint